style.css

```diff
@@ -26,7 +26,7 @@ header {
 .headline {
   background: #1c1a19;
   color: #fff;
-  padding: 130px 25px 145px;
```

Comment:
- [ ] Please stick with the design as much as possible in order to meet the design because fulfilling the requirements helps to make you a good developer and also shows that you are detailed-oriented.


<table>
<tr>
<th align=“middle”>Figma design</th>
<th align=“middle”>Your design</th>
</tr>
<tr>
<td>

![image](https://user-images.githubusercontent.com/66476329/234921426-d0e3f391-64ac-49c1-8c7c-bd5f14397c66.png)


</td>
<td>

![Screenshot from 2023-04-27 21-34-14](https://user-images.githubusercontent.com/66476329/234921264-7ee6d7f5-db21-4376-bd3f-5f7db9cc1bee.png)


</td>
</tr>
</table>